docs-src/content/using-cubic-sdk/streaming.md

```diff
@@ -558,4 +558,343 @@ public static void streamMicrophoneAudio() {
 ```
 {{% /tab %}}
 
+{{% tab "Swift/iOS" %}}
+
+This example uses CubicManager class and its CubicManagerDelegate protocol as a set of instruments for establishing connection to Cubic server, listing models, recording audio stream and performing rrecognition requests. You can use this class in your client view controller or any other object. See the CubicClient class example usage below.
```

Comment:
I think I would probably put the file they bring into their code as a file outside of the docs, with a link to it, and then shorten the example down to just the import and use code.  This seems like a massive wall of code for an example in documentation.

Reply:
I agree. I have replaced the implementation of CubicManager with the GitHub link.

Reply:
I think that will look better.  Thanks.